language/tools/move-cli/src/main.rs

```diff
@@ -198,18 +198,27 @@ pub enum ExperimentalCommand {
 
 fn main() -> Result<()> {
     let move_args = Move::from_args();
+    let mode = Mode::new(move_args.mode);
 
     match &move_args.cmd {
         Command::Compile {
             source_files,
             no_source_maps,
             check,
         } => {
+            let stdlib_mode_interface_dir = mode
```

Comment:
Nit: `mode_interface_dir` might be slightly less confusing, since there is a `stdlib` mode, but `mode` could be (e.g.) `bare` or `diem`?